### PR TITLE
Typos

### PR DESCRIPTION
**Pull request by Pierre-Yves-Gaillard**

### CODE REVIEW

File: basics.tex [hunk 1]
```diff
@@ -1276,7 +1276,7 @@ \section{Cartesian product types}
   Now by induction on $p$ and $q$, we may assume they are reflexivities $\refl a$ and $\refl b$, in which case~\eqref{eq:path-prod-inverse} yields $\refl{(a,b)}$ and then~\eqref{eq:path-prod} returns us to $(\refl a,\refl b)\jdeq (p,q)\jdeq s$.
 \end{proof}
 
-In particular, we have shown that~\eqref{eq:path-prod} has an inverse~\eqref{eq:path-prod-inverse}, which we may denote by
+In particular, we have shown that~\eqref{eq:path-prod} has a quasi-inverse~\eqref{eq:path-prod-inverse}, which we may denote by
```
Comment (mikeshulman) [inline basics.tex]: Hmm.  We defined "a quasi-inverse" to be a function in the other direction *together with* two homotopies.  I would have thought it's okay to say "inverse" when referring only to the inverse function.

Comment (Pierre-Yves-Gaillard) [inline basics.tex]: OK, I agree. I thought the author forgot the "quasi". Sorry!

Comment (mikeshulman) [inline basics.tex]: You can update a pull request by pushing more patches to the same branch.

**Comment by Pierre-Yves-Gaillard** [comment]
Replaced with PR #769.

**Pull request closed without merging**